i18n/locales/en/translations.ts

```diff
@@ -67,69 +67,71 @@ export const translations = {
   },
 
   reaping_rewards: {
-    title: 'Of blocks and miners',
-    subtitle:
-      '"To understand the mystery and magic, you must start at the beginning…"',
-    paragraph_one: `Satoshi, the pseudonymous inventor of bitcoin, also mined the first bitcoin block. He left you a hidden clue in the very first bitcoin transaction that was ever made.`,
-    paragraph_two: `Each block in the bitcoin blockchain contains a set of transactions. A nice way to view these blocks and the things they contain is by using a block explorer.`,
+    title: null,
```

Comment:
Not sure if this is final since this is a Draft pr, but let's remove these both from this translation file and the actual component if they are not being used, otherwise this happens:
<img width="602" alt="image" src="https://user-images.githubusercontent.com/10757768/223984042-001d32b3-667e-4fcc-9d09-1f5db1caddd4.png">

Reply:
Yes, that was my question in the top comment. We may need some flexibility here when adding title and subtitle. Some pages might need it, others don't. Is there a clean way to make this work? Like setting those translation values to null or so?

Reply:
I noticed that the file `Introduction.tsx` while being a template was quite limiting so I went ahead and created a draft PR to generalize the template for these copy PR's it should make it easier to reuse it

Reply:
Thank you @benalleng, I implemented a slightly modified version of your change in this PR, just so I can remove the draft status and it can be reviewed quickly.